cdr/resolver.go

```diff
@@ -114,7 +115,8 @@ type CAADistributedResolver struct {
 	logger      blog.Logger
 }
 
-// New returns a initialized CAADistributedResolver
+// New returns an initialized CAADistributedResolver which requires a M of N
+// quorum to succeed where M = |maxFailures| and N = len(|proxies|)
```

Comment:
Whoops, will fix. The vertical bars are actually a sign I've been working on Chromium too much recently, will remove.